bin/pycbc_live

```diff
@@ -757,6 +760,9 @@ fft.verify_fft_options(args, parser)
 
 if args.output_background is not None and len(args.output_background) != 2:
     parser.error('--output-background takes two parameters: period and path')
+if not args.enable_gracedb_upload and args.enable_single_detector_upload:
```

Comment:
This looks good and is a better way to do it than the one I proposed (which would have been enforcing this silently)